src/ol/controls/RoutingControl.js

```diff
@@ -63,6 +63,9 @@ class RoutingControl extends Control {
       },
     });
 
+    /** @ignore */
+    this.graphs = options.graphs || [];
```

Comment:
Could we integrate the 'osm' graph value by default in the default array ?

Reply:
Good idea! Is done together with JSDoc comment.